src/user/profile/profile-feed.js

```diff
@@ -0,0 +1,249 @@
+        >
+          <Card className="cardElement">
+            <div className="cardTitle">
+              <div className="imageContainer">
+                <Image src={userIcon2} rounded className="userImage" />
+              </div>
+              <div className="titleDetails">
+                <div className="postTitle">
+                  {`${post.userId.name.firstName} ${post.userId.name.lastName}`}
+                </div>
+                <dic className="postDate">{post.createdAt}</dic>
+              </div>
+            </div>
+            <div className="cardContent">{parse(post.content)}</div>
+            <div className="cardTag">
+              <Circle r={5} fill={{ color: "#2409ba" }} /> Post
+            </div>
+          </Card>
+        </div>
+      );
+    });
+
+    const overviewContent = [...posts, ...events];
+
+    return (
+      <div>
+        <div className="feed-posts">
+          <div className="categories">
+            <div className="tab__container">
+              <span className="nav__tab container">
+                <ul className="nav__list__container">
+                  <li
+                    className={
+                      type === "All"
+                        ? "nav__single__tab selected"
+                        : "nav__single__tab"
+                    }
+                    onClick={() => this.handleClick("All")}
+                  >
+                    <FaGlobe className="tab__icon" />
+                    Overview
+                  </li>
+                  <li
+                    className={
+                      type === "Post"
+                        ? "nav__single__tab selected"
+                        : "nav__single__tab"
+                    }
+                    onClick={() => this.handleClick("Post")}
+                  >
+                    <FaComments className="tab__icon" />
+                    Pinned Posts
+                  </li>
+                  <li
+                    className={
+                      type === "Event"
+                        ? "nav__single__tab selected"
+                        : "nav__single__tab"
+                    }
+                    onClick={() => this.handleClick("Event")}
+                  >
+                    <FaCalendar className="tab__icon" />
+                    Events
+                  </li>
+                  <li
+                    className={
+                      type === "ReadMe"
+                        ? "nav__single__tab selected"
+                        : "nav__single__tab"
+                    }
+                    onClick={() => this.handleClick("ReadMe")}
+                  >
+                    <FaStickyNote className="tab__icon" />
+                    Read Me
+                  </li>
+                </ul>
+              </span>
+            </div>
+          </div>
+        </div>
+        <div className="postsContainer">
+          {type === "ReadMe" ? <ReadMe /> : null}
```

Comment:
@AuraOfDivinity No need to fetch again user profile in Readme component just pass from this component and use it as props.